xcell/tests/test_cls.py

```diff
@@ -210,8 +214,8 @@ def test_get_covariance():
     chi2 = dCl.dot(icov).dot(dCl)
     chi2_m = dCl.dot(icov_m).dot(dCl)
 
-    assert np.fabs(chi2/chi2_m) - 1 < 0.01
     shutil.rmtree(tmpdir1)
+    assert np.fabs(chi2/chi2_m-1) - 1 < 0.03
```

Comment:
Why do you need to increase the relative error?

Reply:
Because now we're comparing with the covariance calculated from the estimated C_ell, which is a bit noisy. I can use two different tolerances for the two modes (note also that the previous test was wrong!)

Reply:
you're right! I think we can leave it as is now.